private_chaincode_apps/echo/echo_json.h

```diff
@@ -0,0 +1,28 @@
+/*
+* Copyright 2019 Intel Corporation
+* Copyright IBM Corp. 2018 All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+ */
+
+#pragma once
+
+#include <stdint.h>
+#include <string>
+
+typedef struct echo {
+    std::string echo_string;
+} echo_t;
+
+int unmarshal(echo_t* echo, const char* json_bytes, uint32_t json_len);
```

Comment:
maybe json is not even needed in this echo chaincode

Reply:
probably. but it can be enhanced later